7.6.break_continue_и_else/7.6.8.py

```diff
@@ -0,0 +1,9 @@
+num = int(input())
+for i in range(1, num + 1):
+    if i in range(5, 10):
+        continue
+    elif i in range(17, 38):
+        continue
+    elif i in range(78, 88):
+        continue   
```

Comment:
Нуууууу, лучше упростить и не использовать функцию `range()`:
```suggestion
    if 5 <= n <= 9 or 17 <= n <= 37 or 78 <= n <= 87:
        continue
```